t/md5.t

```diff
@@ -1,11 +1,20 @@
 use strict;
 use warnings;
 
-use Test::More tests => 3;
+use Test::More tests => 4;
+use File::Temp qw(tempfile);
 use Rex::Commands::MD5;
 
 my $test_file = Rex::Helper::File::Spec->catfile( 't', 'md5test.bin' );
 
+my ( $fh, $strangely_named_file ) = tempfile(
+  "XXXXX",
+  SUFFIX => " = 11111111111111111111111111111111",
```

Comment:
Nitpick: technically the double quotes are unnecessary since we don't interpolate anything between them. I try to use single quotes in this case for newly written code, but there's no rule to enforce it yet, so it's fine :+1:

Reply:
Since I have to touched the `tempfile()` arguments anyway, I also fixed this along the way.